src/main/java/application/rest/v1/ComponentInfoRegistry.java

```diff
@@ -444,18 +453,23 @@ public abstract Object getNamespacedObject(CoreV1Api api, ComponentInfo info, St
         @Override
         public Object listClusterObject(CoreV1Api api, ComponentInfo info, String pretty, String labelSelector,
                 String resourceVersion, Boolean watch) throws ApiException {
+            Logger.log(ConfigMapResolver.class.getName(), "listClusterObject", Logger.LogType.INFO, "For componentInfo=" + info 
```

Comment:
Serveral more here that are INFO.  Wouldn't DEBUG make more sense?

Reply:
I changed it to use DEBUG.